neuvector-controller-5.3.yaml

```diff
@@ -38,9 +39,40 @@ pipeline:
       packages: .
       output: controller
 
+  - runs: |
+      # Create Folders
+      mkdir -p ${{targets.contextdir}}/etc
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      mkdir -p ${{targets.contextdir}}/etc/neuvector/templates
+      mkdir -p ${{targets.contextdir}}/etc/tmp
+
+      # Scripts
+      cp scripts/sysctl.conf ${{targets.contextdir}}/etc/
+      cp scripts/teardown.sh ${{targets.contextdir}}/usr/local/bin/
+      cp scripts/runtime-gdb.py ${{targets.contextdir}}/usr/local/bin/
+
+      # Template
+      cp templates/podTemplate.json ${{targets.contextdir}}/etc/neuvector/templates/podTemplate.json
+      wget https://raw.githubusercontent.com/neuvector/manifests/main/build/share/.nvcontainer -P ${{targets.contextdir}}/usr/local/bin/.nvcontainer
```

Comment:
What is this for? It seems like an empty file?

Reply:
They use it as a way of determining whether or not it's running in a container: https://github.com/neuvector/neuvector/blob/490eda9564b36c78f918ab44db21b6de8eba1d0d/share/system/cgroup_linux.go#L112

I would just use `touch` to create the file

Reply:
Thanks for clarifying @EyeCantCU - made a suggestion for you @Pyronewbic